components/permissions/contexts/brave_ethereum_permission_context.cc

```diff
@@ -71,4 +93,76 @@ void BraveEthereumPermissionContext::RequestPermission(
                                            user_gesture, std::move(callback));
 }
 
+// static
+void BraveEthereumPermissionContext::Accept(
```

Comment:
re: comment below, if anything this method would be named `AcceptOrCancel` instead of the PermissionRequestManager method, but I think the convention here is `DecidePermission`

Reply:
https://github.com/brave/brave-core/commit/39529121ec8feba7ceb823c6ba12861e24d5c6ff